LDomadia/c-546-group-19-project
# Issue #60: edit clothes bug

**Also capitalized stats page**

Things to test:
- try adding color patterns in mixed case
- add color patterns in edit
- remove color patterns in edit
- add brand name to a clothing item that doesn't have one
- remove brand name
- ensure that stats are updating consistently and accurately
- try adding duplicate chips to color/patterns and styles
